3.functions/Q5.c

```diff
@@ -5,3 +5,27 @@
     // Example:\n
     // Input: 3, 7, 5\n
     // Output: "The maximum number is 7."\n
+
+#include<stdio.h>
+
+int maxOfThree(int a, int b, int c){
+    if (a > b && a > c)
+        return a;
+    if (b > a && b > c)
+        return b;
+    if (c > a && c > b)
```

Comment:
1. try this code for this input, 10, 10 , 6.
2. do we really need third if in maxOfThree function?